module/library/institutionalAssessment/helper.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * name : helper.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Institutional library helper functionality.
```

Comment:
@aman-tunerlabs - Description should be "Institutional assessment library helper functionality."